static/css/thing.css

```diff
@@ -21,6 +21,23 @@
   background-image: url('../images/unknown-thing.png');
 }
 
+.thing-details-link {
+  box-sizing: border-box;
+  display: block;
+  position: relative;
+  z-index: 10;
+  float: right;
+  background-color: #89b6d6;
+  background-size: 3.2rem;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url('../images/thing-details-link.svg');
+  border: 2px solid white;
```

Comment:
I think 1px might actually look better for such a small circle, even though it's inconsistent with the thing icon.